editor.planx.uk/src/pages/FlowEditor/lib/analytics/types.ts

```diff
@@ -108,6 +109,16 @@ type HelpClick = {
   metadata: HelpClickMetadata;
 };
 
+/**
+ * Captured when a user clicks Save and Return.
+ * The mutation sets the "has_clicked_help: true" when Save and
+ * Return button is clicked
+ */
+
+type SaveClick = {
+  event: "saveClick";
```

Comment:
I think it's worth keeping `metadata: null` here to match other `EventData` interfaces :+1